messages/personal_token/messages.go

```diff
@@ -0,0 +1,10 @@
+package personal_token
+
+const (
+	USAGE                      = "personal-token"
+	SHORT_DESCRIPTION_DESCRIBE = "Describe a Personal Token"
+	LONG_DESCRIPTION_DESCRIBE  = "Describe a Personal Token to be used for authentication and security"
+	ASK_PERSONAL_TOKEN_ID      = "Enter the ID of the personal token is linked to:"
```

Comment:
"Enter the Personal Token's ID:"


-------
this is to follow the pattern of the other describe commands